web-admin/src/routes/[organization]/[project]/[dashboard]/+page.svelte

```diff
@@ -42,21 +47,60 @@
   }
 
   async function getDashboardsAndInvalidate() {
-    return invalidateProjectQueries(
-      queryClient,
-      await getDashboardsForProject($project.data)
-    );
+    const dashboardListings = await getDashboardsForProject($project.data);
+    const dashboardNames = dashboardListings.map((listing) => listing.name);
+    return invalidateProjectQueries(queryClient, dashboardNames);
```

Comment:
Since we now have the file and catalog entry we need to update `invalidateProjectQueries` to invalidate them.

Reply:
This is needed since there could be stale data @ericpgreen2